actors/miner/tests/prove_commit_sector_3_test.rs

```diff
@@ -489,6 +489,418 @@ fn rejected_notification_dropped() {
     verify_weights(&rt, &h, snos[1], 0, piece_size);
 }
 
+#[test]
+fn aggregate_proof_min_sectors() {
+    let (h, mut rt) = setup_basic();
+    let piece_size = h.sector_size as u64;
+    let sector_count =
+        fil_actors_runtime::runtime::policy_constants::MIN_AGGREGATED_SECTORS as usize;
+
+    let piece_config = [piece_size];
+    let piece_configs: Vec<&[u64]> = (0..sector_count).map(|_| &piece_config[..]).collect();
+    let precommits = precommit_sectors(&mut rt, &h, &piece_configs);
+    let snos: Vec<SectorNumber> =
+        precommits.iter().map(|pci: &SectorPreCommitInfo| pci.sector_number).collect();
+
+    let manifests: Vec<_> = snos
+        .iter()
+        .enumerate()
+        .map(|(i, sno)| {
+            // Mix of CC sectors and sectors with deals/allocations
+            if i == 0 {
+                // CC sector
+                make_activation_manifest(*sno, &[(piece_size, 0, 0, 0)])
+            } else if i == 1 {
+                // With allocation only
+                make_activation_manifest(*sno, &[(piece_size, CLIENT_ID, 1000 + i as u64, 0)])
+            } else {
+                // With deal and allocation
+                make_activation_manifest(
+                    *sno,
+                    &[(piece_size, CLIENT_ID, 1000 + i as u64, 2000 + i as u64)],
+                )
+            }
+        })
+        .collect();
+
+    let cfg = ProveCommitSectors3Config::default();
+    let (result, claims, notifications) =
+        h.prove_commit_sectors3(&rt, &manifests, true, true, true, cfg).unwrap();
+
+    assert_commit_result(&vec![ExitCode::OK; sector_count], &result);
+
+    // Verify sectors were committed
+    for (i, sno) in snos.iter().enumerate() {
+        let sector = h.get_sector(&rt, *sno);
+        assert_eq!(sector.sector_number, *sno);
+
+        if i == 0 {
+            // CC sector - only deal weight
+            verify_weights(&rt, &h, *sno, piece_size, 0);
+        } else if i == 1 {
+            // Allocation only - only verified weight
+            verify_weights(&rt, &h, *sno, 0, piece_size);
+        } else {
+            // Deal + allocation - only verified weight (allocation takes precedence)
```

Comment:
nit: comment is somewhat inaccurate.  Remove "(allocation takes precedence)". Deal is orthogonal to allocation.

Reply:
Done.